test/unit/DCAStrategies/dca-strategies-permissions-handler.spec.ts

```diff
@@ -0,0 +1,175 @@
+import { expect } from 'chai';
+import { ethers } from 'hardhat';
+import { DCAStrategiesPermissionsHandlerMock__factory, DCAStrategiesPermissionsHandlerMock } from '@typechained';
+import { constants, wallet, behaviours } from '@test-utils';
+import { given, then, when, contract } from '@test-utils/bdd';
+import { snapshot } from '@test-utils/evm';
+import { Permission } from 'utils/types';
+import { TransactionResponse } from '@ethersproject/abstract-provider';
+// import { getInstancesOfEvent, readArgFromEventOrFail } from '@test-utils/event-utils';
+import { Wallet } from '@ethersproject/wallet';
+import { BigNumber } from '@ethersproject/bignumber';
+import { _TypedDataEncoder } from '@ethersproject/hash';
+import { fromRpcSig } from 'ethereumjs-util';
+import { BigNumberish } from 'ethers';
+
+contract('DCAStrategiesPermissionsHandler', () => {
+  const NFT_NAME = 'Mean Finance - DCA Strategy Position';
+  const NFT_SYMBOL = 'MF-DCA-P';
+  let DCAStrategiesPermissionsHandlerMockFactory: DCAStrategiesPermissionsHandlerMock__factory;
+  let DCAStrategiesPermissionsHandlerMock: DCAStrategiesPermissionsHandlerMock;
```

Comment:
This is just a recommendation, feel free to ignore it
```suggestion
  let handler: DCAStrategiesPermissionsHandlerMock;
```